controllers/metrics.go

```diff
@@ -151,6 +158,15 @@ func collectMetrics(ch chan<- prometheus.Metric, cluster *fdbv1beta2.FoundationD
 		addGauge(descProcessGroupMarkedRemoval, float64(removals[pclass]), string(pclass))
 		addGauge(descProcessGroupMarkedExcluded, float64(exclusions[pclass]), string(pclass))
 	}
+
+	counts, err := cluster.GetProcessCountsWithDefaults()
+	if err != nil {
+		return
```

Comment:
Can we add a log statement here. Just to know the error and for bookkeeping that we hadn't calculated the metrics for desired number of processes.

Reply:
Adding a logger here requires some refactoring. I can open an issue for that to add a logger to the metrics collector.